geoips/interfaces/yaml_based/products.py

```diff
@@ -95,19 +95,6 @@ class ProductsInterface(BaseYamlInterface):
     name = "products"
     validator = ProductsPluginValidator()
 
-    @staticmethod
-    def _create_plugin_cache_names(yaml_plugin):
-        """Create a plugin name for cache storage.
-
```

Comment:
@evrose54 @jsolbrig I removed this unused method in product interface.

Reply:
See: https://github.com/NRLMMD-GEOIPS/geoips/pull/380/files
Resolve this conversation if you are good with those changes.